site/_contributors/addy.txt

```diff
@@ -0,0 +1,6 @@
+---
+weight: 1
+name: Addy
+image: /assets/images/addy.png
```

Comment:
These look great. The only comment I have would echo the earlier one about image compression :)